src/apps/persistence/jobs.clj

```diff
@@ -207,14 +207,20 @@
       first
       :status))
 
+(defn- submission->save-format
+  [submission]
+  (-> submission
+      (dissoc :job_config)
+      cheshire/encode))
+
 (defn save-job
   "Saves information about a job in the database."
   [{system-id :system_id username :username :as job-info} submission]
   (-> (select-keys job-info job-fields)
       (assoc :job_type_id (job-type-id-from-system-id system-id)
              :user_id     (get-user-id username))
       remove-nil-values
-      (save-job-with-submission (cheshire/encode submission))))
+      (save-job-with-submission (submission->save-format submission))))
```

Comment:
This new format function will remove the new `job_config` key from the submission before saving to the db.